src/main/scala/se/nimsa/sbx/seriestype/SeriesTypeProtocol.scala

```diff
@@ -0,0 +1,20 @@
+package se.nimsa.sbx.seriestype
+
+object SeriesTypeProtocol {
+
+  import se.nimsa.sbx.model.Entity
+  
+  case class SeriesType(id: Long, name: String) extends Entity
+  
+  case class SeriesTypeRule(id: Long, seriesTypeId: Long) extends Entity
```

Comment:
Fattar inte :-) 
En regel säger till exempel "PatientName (0010:0010) ska vara "Karl". Men i case klassen SeriesTypeRule finns ju ingen plats för själva värdet "Karl", inte i SeriesTypeRuleAttribute heller. Var kommer detta in?